rspace-bench/src/main/scala/coop/rchain/rspace/bench/wide/ReplayWideBench.scala

```diff
@@ -0,0 +1,71 @@
+package coop.rchain.rspace.bench.wide
+
+import coop.rchain.rspace.bench._
+import coop.rchain.rholang.interpreter.ChargingReducer
+import coop.rchain.rspace.bench.EvalBenchStateBase
+import java.io.{FileNotFoundException, InputStreamReader}
+import java.nio.file.{Files, Path}
+import java.util.concurrent.TimeUnit
+
+import coop.rchain.catscontrib.TaskContrib._
+import coop.rchain.crypto.hash.Blake2b512Random
+import coop.rchain.models.Par
+import coop.rchain.rholang.interpreter.accounting.Cost
+import coop.rchain.rholang.interpreter.{Interpreter, Runtime}
+import coop.rchain.shared.StoreType
+import monix.eval.Task
+import monix.execution.Scheduler
+import org.openjdk.jmh.annotations._
+import org.openjdk.jmh.infra.Blackhole
+
+import scala.concurrent.Await
+import scala.concurrent.duration.{Duration, _}
+
+class ReplayWideBench {
+
+  @Benchmark
+  @BenchmarkMode(Array(Mode.SingleShotTime))
+  @OutputTimeUnit(TimeUnit.MILLISECONDS)
+  @Fork(value = 1)
+  @Threads(1)
+  @Warmup(iterations = 2)
+  @Measurement(iterations = 5)
+  def wideReduce(bh: Blackhole, state: ReplayFineBenchState): Unit = {
+    implicit val scheduler = state.scheduler
+    val result             = state.runReplayTask.unsafeRunSync
+    assert(result.isEmpty)
+    bh.consume(result)
+  }
+}
+
+@State(Scope.Benchmark)
+class ReplayFineBenchState extends ReplayWideBenchState {
+  override def createRuntime(): Runtime = Runtime.create(dbDir, mapSize, StoreType.FineGrainedLMDB)
+}
+
+abstract class ReplayWideBenchState extends WideBenchBaseState {
+
+  implicit val rand: Blake2b512Random = Blake2b512Random(128)
+
+  var runReplayTask: Task[Vector[Throwable]] = Task.now(Vector.empty)
+
+  @Setup(value = Level.Iteration)
+  override def doSetup(): Unit = {
+    super.doSetup()
+
+    {
```

Comment:
Looks like '{' doesn't required here and left from old code?

Reply:
Not really, local block was there so it's possible to have 2 `val checkpoint`'s, but I renamed the first one and removed the local block to avoid confusion